plugins/devtools-common/src/permissions.ts

```diff
@@ -47,3 +47,11 @@ export const devToolsExternalDependenciesReadPermission = createPermission({
   name: 'devtools.external-dependencies',
   attributes: { action: 'read' },
 });
+
+/**
+ * @public
+ */
+export const devToolsTasksReadPermission = createPermission({
```

Comment:
Should we also have a update permission for the Trigger? That would give more fine control over this.

Reply:
I originally had this but decided that if the user can read the task, it can also be triggered. I don't know if that should be separated or not

Reply:
Yeah, that's fair